frontend/src/pages/admin/pending_users/PendingUsers.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+
+import { Helmet } from "react-helmet";
+import { makeStyles } from "@material-ui/core/styles";
+import { useDispatch } from "react-redux";
+import MaterialTable from "@material-table/core";
+import { Typography } from "@material-ui/core";
+
+import PageContainer from "../../../components/PageContainer";
+import { withAuthorization } from "../../../components/HOC";
+import { getUsers } from "../../../services/users";
+import { getRoles } from "../../../services/roles";
+import LoadingContainer from "../../../components/LoadingContainer";
+import { openAlert } from "../../../actions";
+import { TableIcons } from "../../../components/Icons";
+
+const useStyles = makeStyles(() => ({
+  columnTitle: {
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+}));
+
+const PendingUsers = () => {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const [state, setState] = useState({
+    loading: true,
+    users: [],
+  });
+
+  useEffect(() => {
+    const loadUsers = async () => {
+      const roles_res = await getRoles("Pending");
+
+      if (roles_res.ok) {
+        if (roles_res.data.roles.length === 0) {
+          dispatch(openAlert("Error: Pending role not found"));
+          setState({
+            ...state,
+            loading: false,
+          });
+
+          return;
+        }
+
+        const user_res = await getUsers(roles_res.data.roles[0]._id);
+
+        if (user_res.ok) {
+          setState({
+            ...state,
+            loading: false,
+            users: user_res.data.users,
+          });
+        } else {
+          dispatch(openAlert(`Error: ${user_res.data.message}`));
+        }
+      } else {
+        dispatch(openAlert(`Error: ${roles_res.data.message}`));
+      }
+    };
+
+    if (state.loading) {
+      loadUsers();
+    }
+  }, []);
+
+  return (
+    <>
+      <Helmet>
+        <title>Pending Users — TSE Oktavian</title>
+      </Helmet>
+      <PageContainer>
+        <LoadingContainer loading={state.loading}>
+          <MaterialTable
+            title="Pending Users"
+            icons={{ ...TableIcons, Filter: TableIcons.Search }}
+            options={{
+              filtering: true,
```

Comment:
I'm wondering if its useful to have the search bar at the top of the table if we already have the filtering for each column, or vice versa. I fell like we only need one of them.  If you want to get rid of searching in each column you can set `filtering: false`, if you want to remove the search bar at top then set `search: false` in the options. Not sure if this is something we want to address, @hahsu what do you think?

Reply:
Yeah, I agree it's a bit redundant, and since the designs only have the search for each column, let's match that and set `filtering: false` for the table.

Reply:
I set `search: false`, which removed the search from the top of the table.

![image](https://user-images.githubusercontent.com/36427506/119218949-ab32e700-ba97-11eb-90d7-14f5a4bac1b3.png)